migration-playbook.yml

```diff
@@ -35,11 +35,11 @@
   roles:
     - { role: setup-local }
     - { role: download-site, tags: [download-site] }
-    - { role: check-php, tags: [download-site,check-php] }
+    - { role: css-injector, tags: [download-site,css-injector] }
+    - { role: check-php, tags: [download-site,css-injector,check-php] }
```

Comment:
Should `css-injector` be a tag here? What other roles should get the `css-injector` tag?

Reply:
These tags all need to be reviewed.

Reply:
Agreed!  Have you been using them?  Perhaps we can do without them entirely?  If you are using them, maybe trim them down?  I've been pretty much just restarting migrations from scratch when something goes wrong.

Reply:
I’ve been running everything from scratch, too. Sounds like this needs to be its own ticket. That’s a good enough answer for me.

I moved things around enough in this PR that I don’t think it should be held up on tags.

Reply:
Here you go: https://stanfordits.atlassian.net/browse/SITES-594.  Phew.  I'm seeing just how much easier it is to create tickets than to complete them.  That seems like a dangerous dichotomy.